README.md

```diff
@@ -19,6 +27,16 @@ npm test
 ```bash
 zip -r cl-synth-index-cc.zip .
 ```
+## Run with Docker
+
+```bash
+docker build . -t synth-index-cc-adapter
+docker run -itd \
+    -p 8080:8080 \
+    -e API_KEY="Your_cryptocompare_API_key" \
+    --name "YOUR_CUSTOM_CONTAINER_NAME" \
```

Comment:
I'd prefer if we leave out things like naming the container, just for consistency between repos.

Reply:
> 
> 
> See comments regarding Yarn and the readme.

Hi. Thanks for pointing out. Honestly didn't see it but with yarn it looks good indeed. Will test the app first and then update the PR.

Reply:
I'm not getting that warning locally, but through Docker I get it. Should be fine though.

I was however not successful in running the Dockerfile in this PR (with npm replaced with Yarn). Changing docker image to "current" seemed to fix it, as well as removing redundant commands:

```Dockerfile
FROM node:current

WORKDIR /synth-index-cc-adapter
ADD . .

RUN yarn install

CMD node ./app.js
```

Reply:
I tested them with both base images, both are working.
I recommend the "alpine" version since it's a lot smaller than the one tagged "current".
```
Current:
REPOSITORY                TAG                 IMAGE ID            CREATED             SIZE
synth-index-cc-adapter    latest              a453096235dc        46 seconds ago      1.37GB

Alpine:
REPOSITORY                TAG                 IMAGE ID            CREATED             SIZE
synth-index-cc-adapter    latest              6838bdf4cd0d        7 minutes ago       566MB
```

What error you have when using this Dockerfile?

```
FROM node:alpine

WORKDIR /synth-index-cc-adapter
ADD . .

RUN yarn install

CMD node ./app.js
```

I was searching through the repositories in order to find similar setups but I could only find this one containing a Dockerfile and it's using also alpine.

Reply:
Yep, disregard what I said about alpine not working – must have had an older version.

Reply:
Great, this should do it then.